### ISSUE TITLE
Remix started with root folder opened

### ISSUE THREAD

**Issue by daithi-coombes** [ethereum/remix-desktop#78]
os:
```
M1 Silicon
Darwin Kernel Version 20.5.0:
root:xnu-7195.121.3~9/RELEASE_ARM64_T8101
arm64
```
Remix Desktop: `Version 1.3.2 (1.3.2.548)`

Haven't been able to recreate, yet. When I opened remix-desktop it opened at the root folder, given it was on a mission critical machine this is worrying.

Is there any way or code flow that would allow the IDE to open with the root `/` folder?

**Comment by yann300** [comment]
thanks for the report and sorry for the late answer, was it the root folder or the home folder? how did you start it?
Thanks

**Comment by rsmets** [comment]
It started in the root folder for me too. But oddly it won't open any of my files. Stuck "Loading...". On MacOS as well.

**Comment by daithi-coombes** [comment]
it stopped happening for me when I rebooted. But just now an important macos upgrade caused rosetta to break anything with electronJS, so everything had to be re-installed and this came back. After re-install the issue came back but with home `$HOMEDIR` instead of root folder, which is improvement 👍 

steps to reproduce
 - re-open the `dmg` installer
 - drag to applications and replace (I replaced 1.3.2 with 1.3.3)
 - re-open remix ide
 
One difference with 1.3.3 than the op version, it opens on my home dir with hidden files shown! as the workspace and can browse everything.

Maybe this is expected result but I'd expect the IDE to only have knowledge of 'workspaces' or dirs that I allow it
 
 Its an improvement from the `/` folder but I'd expect the IDE to only access folders I give it and start with default only 🤔
